scale/storage/brokers/broker.py

```diff
@@ -71,7 +71,7 @@ def delete_files(self, volume_path, files):
         :type files: [:class:`storage.models.ScaleFile`]
         """
 
-        pass
+        raise NotImplementedError
```

Comment:
I think it should be `raise NotImplementedError()` here and everywhere else in the file.